tests/basic/pool_function_test.go

```diff
@@ -176,232 +177,232 @@ var _ = Describe("{PoolExpandSmoky}", func() {
-//			// Ensure poolToResize is not nil
-//			dash.VerifyFatal(poolToResize != nil, true, "Pool is up and running after restart")
-//			verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
-//		})
-//	})
-//
-//	It("Initiate pool expansion using add-drive and restart PX", func() {
-//		StartTorpedoTest("PoolExpandAddDiskAndPXRestart",
-//			"Initiate pool expansion using add-drive and restart PX", nil, testrailID)
-//
-//		Step("Select a pool that has I/O and expand it by 100 GiB with add-disk type. ", func() {
-//			originalSizeInBytes = poolToResize.TotalSize
-//			targetSizeInBytes = originalSizeInBytes + 100*units.GiB
-//			targetSizeGiB = targetSizeInBytes / units.GiB
-//			log.InfoD("Current Size of the pool %s is %d GiB. Trying to expand to %v GiB with type add-disk",
-//				poolIDToResize, poolToResize.TotalSize/units.GiB, targetSizeGiB)
-//			triggerPoolExpansion(poolIDToResize, targetSizeGiB, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK)
-//		})
-//
-//		Step("Wait for expansion to start and reboot node", func() {
-//			err := WaitForExpansionToStart(poolIDToResize)
-//			log.FailOnError(err, "Timed out waiting for expansion to start")
-//			err = Inst().V.RestartDriver(*storageNode, nil)
-//			log.FailOnError(err, fmt.Sprintf("Error restarting px on node [%s]", storageNode.Name))
-//			err = Inst().V.WaitDriverUpOnNode(*storageNode, addDriveUpTimeOut)
-//			log.FailOnError(err, fmt.Sprintf("Timed out waiting for px to come up on node [%s]", storageNode.Name))
-//		})
-//
-//		Step("Ensure pool has been expanded to the expected size", func() {
-//			resizeErr := waitForOngoingPoolExpansionToComplete(poolIDToResize)
-//			dash.VerifyFatal(resizeErr, nil, "Pool expansion does not result in error")
-//			verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
-//		})
-//	})
-//})
-//
+var _ = Describe("{PoolExpandRejectConcurrent}", func() {
+	BeforeEach(func() {
+		contexts = scheduleApps()
+	})
+
+	JustBeforeEach(func() {
+		poolIDToResize = pickPoolToResize()
+		log.Infof("Picked pool %s to resize", poolIDToResize)
+		poolToResize = getStoragePool(poolIDToResize)
+		resizeErr := waitForOngoingPoolExpansionToComplete(poolIDToResize)
+		dash.VerifyFatal(resizeErr, nil, "Previous pool expansion(s) should not result in error")
+		storageNode, err = GetNodeWithGivenPoolID(poolIDToResize)
+		log.FailOnError(err, "Failed to get node with given pool ID")
+	})
+
+	JustAfterEach(func() {
+		AfterEachTest(contexts)
+	})
+
+	AfterEach(func() {
+		appsValidateAndDestroy(contexts)
+		EndTorpedoTest()
+	})
+
+	// test resizing all pools on one storage node concurrently and ensure only the first one makes progress
+	It("Select all pools on a storage node and expand them concurrently. ", func() {
+		// TestRail:https://portworx.testrail.net/index.php?/tests/view/34542836&group_by=cases:custom_automated&group_order=desc&group_id=2
+		StartTorpedoTest("PoolExpandRejectConcurrent",
```

Comment:
plz move this to justbeforeeach or BeforeEach which ever executes first

Reply:
actually i don't own this test,
i have just uncommented those

we will raise independent bugs for each test case if failures or improvements found and assign to test owner